### PR TITLE
Bug 1170907 - Add a basic reading list test

### PR DESCRIPTION
**Pull request by wesj**

### CODE REVIEW

**Comment by wesj** [comment]
lltm

File: UITests/ReadingListTest.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```
Comment (thebnich) [inline UITests/ReadingListTest.swift]: Nit: `*`

File: UITests/ReadingListTest.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+class ReadingListTests: KIFTestCase, UITextFieldDelegate {
+    private var webRoot: String!
+
+    override func setUp() {
+        webRoot = SimplePageServer.start()
+    }
+
+    /**
+    * Tests basic page navigation with the URL bar.
+    */
```
Comment (thebnich) [inline UITests/ReadingListTest.swift]: Wrong comment (and `*`s)

File: UITests/ReadingListTest.swift (hunk 1 above)
Comment (wesj) [inline UITests/ReadingListTest.swift]: I do this every time just for you.

**Pull request merged**